src/eredis.erl

```diff
@@ -30,11 +32,24 @@
 start_link() ->
     start_link("127.0.0.1", 6379).
 
--spec start_link(Host::list(), Port::integer()) -> {ok, Pid::pid()} | {error, Reason::term()}.
+%% Poolboy needs start_link/1
+-spec start_link([option() | {host, host()} | {port, inet:port_number()}]) ->
+          {ok, pid()} | {error, Reason :: term()}.
```

Comment:
space before/after ::

Reply:
I just followed the style of surrounding code, but sure, I'll add spaces.

Reply:
I meant there was extra spaces, i.e `Reason :: term()` chahe to `Reason::term()`

Reply:
Ah, OK